lib/validate_config.js

```diff
@@ -0,0 +1,13 @@
+var CLIEngine = require("eslint").CLIEngine
+  , cli = new CLIEngine()
+  , fs = require("fs");
+
+module.exports = function(configPath) {
+  if (configPath) {
```

Comment:
@wfleming yeah - if you've specified a specific config file this check would pass, but a later one would error.